source/particles/particle_handler.cc

```diff
@@ -939,52 +1030,65 @@ namespace Particles
     // processes around (with an invalid cell).
 
     std::vector<particle_iterator> particles_out_of_cell;
-    particles_out_of_cell.reserve(n_locally_owned_particles());
+
+    particles_out_of_cell.reserve(n_locally_owned_particles() / 4);
```

Comment:
Divided by 4?

Reply:
Just a guess for how many particles need sorting into a different cell. I added documentation.